client/src/Components/Layouts/Captains/EditCaptain/index.jsx

```diff
@@ -0,0 +1,16 @@
+
+import React, { Component } from 'react';
+import CollectionsPage from './editCaptain';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class EditCaptain extends Component {
```

Comment:
@naremanhilles I have to agree with eslint here 😅.  It's not super urgent, but it would be a little more efficient to have this small component be a functional component instead of a class component.  Then the browser doesn't have to use memory to keep track of the class instance.

Reply:
haha, thanks ✨eslint✨